src/services/account-services/list-client-ids-released-for-console-service.js

```diff
@@ -0,0 +1,19 @@
+import { AxiosHttpClientAdapter } from '../axios/AxiosHttpClientAdapter'
+import { getEnvironment } from '@/helpers/get-environment'
+
+export const listClientIdsReleasedForConsoleService = async () => {
+  const httpResponse = await AxiosHttpClientAdapter.request({
+    url: `/allowed-accounts`,
+    method: 'GET'
+  })
+  return adapt(httpResponse)
+}
+
+const adapt = (httpResponse) => {
+  const environment = getEnvironment()
+
+  if (environment === 'production') {
+    return httpResponse?.body?.production?.client_ids ?? []
+  }
+  return httpResponse?.body?.stage?.client_ids ?? []
+}
```

Comment:
Sugestão de melhoria na legibilidade.

```suggestion
const adapt = (httpResponse) => {
  const { body } = httpResponse || {};
  const { production, stage } = body || {};
  const environment = getEnvironment();

  const clientIdsPath = environment === 'production' ? production : stage;
  return clientIdsPath?.client_ids ?? [];
}
```